.gitignore

```diff
@@ -6,10 +6,8 @@ include
 lib
 distribute.egg-info
 setuptools.egg-info
-.coverage
-.tox
 *.egg
 *.py[cod]
 *.swp
 *~
-.hg*
+/.*
```

Comment:
This will also ignore the .gitignore, .hgignore, .hgtags, and .travis.yml files, which need to be included.